install/upgrade-clickhouse.sh

```diff
@@ -17,8 +17,21 @@ if [[ -n "$(docker volume ls -q --filter name=sentry-clickhouse)" ]]; then
   # Wait for clickhouse
   wait_for_clickhouse
 
-  # In order to get to 23.8, we need to first upgrade go from 21.8 -> 22.8 -> 23.3 -> 23.8
+  # In order to get to 23.8, we need to first upgrade go from 20.3 -> 21.8 -> 22.8 -> 23.3 -> 23.8
```

Comment:
Feels like it might be time to codify this information in a machine-readable format which would allow doing some nice things programmatically. WDYT?